{"name": "Тест авторизации с неверным логином и паролем", "status": "passed", "description": "Получение ошибки. Авторизация не выполнена", "start": 1719817077282, "stop": 1719817077517, "uuid": "487d828a-1ec7-44ef-be0a-196e9d032383", "historyId": "7d8ee533535a2ed61f3922b2354a854f", "testCaseId": "7d8ee533535a2ed61f3922b2354a854f", "fullName": "tests.test_api_login_user.TestLoginUser#test_login_user_incorrect_email_and_password_error", "labels": [{"name": "feature", "value": "Проверка авторизации пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "pc"}, {"name": "thread", "value": "5392-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api_login_user"}]}